spec/models/shipment_spec.rb

```diff
@@ -1,5 +1,17 @@
 require "rails_helper"
 
 RSpec.describe Shipment, type: :model do
-  # write your tests here
+  let(:shipment) {FactoryBot.create(:shipment)}
```

Comment:
👍🏾 
Could do this too

```suggestion
  let(:shipment) { create(:shipment) }
```